src/components/SourceFooter.js

```diff
@@ -13,10 +13,10 @@ const { shouldShowFooter, shouldShowPrettyPrint } = require("../utils/editor");
 require("./SourceFooter.css");
 
 function debugBtn(onClick, type, className = "active", tooltip) {
-  className = `${type} ${className}`;
-  return dom.span(
+  className = `${type} ${className} source-footer__command-bar__action`;
+  return dom.button(
     { onClick, className, key: type },
-    Svg(type, { title: tooltip })
+    Svg(type, { title: tooltip, "aria-label": tooltip })
   );
```

Comment:
thanks for helping us make the buttons more accessible. I personally would like to see more of our buttons become `button` elements.